desktop/bin/after_sign_hook.js

```diff
@@ -38,6 +38,7 @@ module.exports = async function ( context ) {
 		appleId: NOTARIZATION_ID,
 		appleIdPassword: NOTARIZATION_PWD,
 		ascProvider: NOTARIZATION_ASC_PROVIDER,
+		teamId: NOTARIZATION_ASC_PROVIDER,
```

Comment:
We'll want to update how we notarize to drop the using a user account, but let's focus on getting the build out first...

Reply:
Internal ref pdnsEh-1Q1-p2